.jenkins/pytorch/build.sh

```diff
@@ -310,3 +310,13 @@ if [[ "${BUILD_ENVIRONMENT}" == *xla* ]]; then
   popd
   assert_git_not_dirty
 fi
+
+if [[ "$BUILD_ENVIRONMENT" ==  *coverage* ]]; then
```

Comment:
This condition would never be executed, as there are no build job which name contains the word `coverage` defined in https://github.com/pytorch/pytorch/blob/master/.circleci/cimodel/data/pytorch_build_definitions.py

Reply:
I fixed it by adding a new `build` job: `pytorch_linux_bionic_py3_8_gcc9_coverage_build` in `config.yml`. Not sure if it is the correct way.

Reply:
That should work, but adding a new build *just* for coverage seems a bit overkill.

Is there a way to piggy-back this onto a separate build so we don't have to add a completely new configuration?

Reply:
Yeah, it takes 2.5 hours to run the new build. Now I revise the existing `pytorch_linux_bionic_py3_8_gcc9_build` to `pytorch_linux_bionic_py3_8_gcc9_coverage_build`. Does this make sense?